install.sh

```diff
@@ -604,6 +604,11 @@ install_werf_system_dependencies() {
     return 0
   fi
 
+  VERSION_ID=$(lsb_release -r | cut -f2)
+  if [[ "$VERSION_ID" == "18.04" ]]; then
```

Comment:
You should never do it like this. What if this is Ubuntu 18.10 or 17.10? There is `compare_versions()` function exactly for this, use it. I see there are more places with checks like this, all of them should be replaced with `compare_versions()`.